internal/node_install.bzl

```diff
@@ -136,7 +136,8 @@ def node_repositories(package_json):
 
   _yarn_repo(name = "yarn", package_json = package_json)
 
-  npm_install(
+  yarn_install(
       name = "build_bazel_rules_nodejs_rollup_deps",
       package_json = "@build_bazel_rules_nodejs//internal/rollup:package.json",
+      yarn_lock = "@build_bazel_rules_nodejs//internal/rollup:yarn.lock",
```

Comment:
can we make it work with only a lock file? or does yarn require we keep a pair of files and pass them both?

obviously for our purposes we don't care about `package.json`
maybe users would be confused though, even if it were possible.

Reply:
Both files are needed for yarn.